echo.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strings"
+)
+
+func check(e error) {
+	if e != nil {
+		panic(e)
+	}
+}
+func parserAndPrint(input string) {
+	words := strings.Fields(input)
+	var err error
+	if len(words) == 0 || words[0] != "echo" {
+		err = fmt.Errorf("error: command must start with 'echo'")
+	}
+	if len(words) == 3 {
+		fmt.Print(words[2])
+	} else if len(words) == 2 {
+		fmt.Println(words[1])
+	} else {
+		err = fmt.Errorf("error: incorrect number of arguments")
+	}
+	check(err)
+
+}
+func main() {
```

Comment:
you won't be able to run this project, there can't be more than one main function in a package
you can either split them into separate packages, or just add one main function (in a main.go file) that handles all the commands as [subcommands](https://gobyexample.com/command-line-subcommands)